confgenerator/testdata/valid/linux/logging-no_conf/input.yaml

```diff
@@ -12,16 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-metrics:
-  receivers:
-    hostmetrics:
-      type: hostmetrics
-      collection_interval: 60s
-  exporters:
-    google:
-      type: google_cloud_monitoring
+logging:
```

Comment:
Should this test case be renamed to "disable_all_logging" instead? Same for the windows test.

Reply:
I was hesitant to rename `testdata` folder files because it could cause rebasing conflicts for others. Will rename after the waves of GA PRs pass.